firefox-ios/firefox-ios-tests/Tests/XCUITests/BrowsingPDFTests.swift

```diff
@@ -28,7 +28,7 @@ class BrowsingPDFTests: BaseTestCase {
     // https://testrail.stage.mozaws.net/index.php?/cases/view/2307117
     // Disabled due to link not loading
     func testOpenLinkFromPDF() throws {
-        throw XCTSkip("Link inside pfd is not loading anymore")
+        // throw XCTSkip("Link inside pdf is not loading anymore")
```

Comment:
I tried this locally and seems to be working for me, could please double check and enable it if that works? since we are looking at this test suite.. 
Thanks!

Reply:
🎉 Yes, this test passes on my end. Note that the URL is from a website we don't control.

Oops. I hit merge before I enable this test. I opened a new PR to enable this test: https://github.com/mozilla-mobile/firefox-ios/pull/19857